integrations/block-data-api.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Automattic\VIP\Integrations;
+
+/**
+ * Loads VIP Block Data REST API.
+ *
+ * @private
+ */
+class BlockDataApi extends Integration {
+	/**
+	 * Applies hooks to load Block Data API plugin.
+	 *
+	 * @private
+	 */
+	public function load( array $config ): void {
+		add_action( 'plugins_loaded', function() {
```

Comment:
One slight worry I have is that we'll forget to include this for a future integration and end up breaking some sites. We could handle this elsewhere, but I worry that could be hard to detangle in the future so let's keep it as-is for now. Let's just document it.

```suggestion
		// Wait until plugins_loaded to give precedence to the plugin in the customer repo
		add_action( 'plugins_loaded', function() {
```